<launch>

<node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen">
    <param name="port" value="/dev/ttyACM0" />
    <param name="baud" value="57600" />
</node>
  <!-- Node 1 -->
  <node pkg="dynamixel_sdk_examples" type="more_motors_node" name="more_motors_node" output="screen" />

  <!-- Node 2 -->
  <node pkg="dynamixel_sdk_examples" type="move_down_grasp_node" name="move_down_grasp_node" output="screen" />
</launch>